docs/spec/api.md.tmpl

```diff
@@ -1000,8 +1000,12 @@ to last response or be fully omitted, depending on the server implementation.
 It may be necessary to list all of the tags under a given repository. The tags
 for an image repository can be retrieved with the following request:
 
-    GET /v2/<name>/tags/list
+    GET /v2/<name>/tags/_list
```

Comment:
This doesn't match was was added above. Please make sure to regenerate from the template.